libraries/provision/provision_cluster.py

```diff
@@ -122,12 +130,7 @@ def provision_cluster(cluster_config, couchbase_server_config, sync_gateway_conf
     sync_gateway_build = None
 
     if opts.sync_gateway_version is not None:
-        version_build = opts.sync_gateway_version.split("-")
-        if len(version_build) != 2:
-            print("Make sure the sync_gateway version follows pattern: 1.2.3-456")
-            sys.exit(1)
-        sync_gateway_version = version_build[0]
-        sync_gateway_build = version_build[1]
+        sync_gateway_version, sync_gateway_build = split_sync_gateway_version(opts.sync_gateway_version)
```

Comment:
You can use https://github.com/couchbaselabs/mobile-testkit/blob/master/keywords/utils.py#L68-L71